modules/react/tokens/README.md

```diff
@@ -188,8 +188,8 @@ Variants only come their augmenting styles and a base type object is required.
 
 | Hierarchy Levels |
 | ---------------- |
-| `dataViz1`       |
-| `dataViz2`       |
+| `brand1`       |
+| `brand2`       |
```

Comment:
There's a disclaimer about the new type hierarchy further down in the README that no longer makes sense (references Labs Type back when it was in Labs/Core/Type) -- not a big deal, though, since I'm guessing you're planning on updating the docs anyway in #1078.

Reply:
But you already knew that :) (I see Alex already pointed this out in his comment earlier)